ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketRequestHandlerTest.java

```diff
@@ -57,13 +62,17 @@
   private Vertx vertx;
   private WebSocketRequestHandler handler;
   private JsonRpcMethod jsonRpcMethodMock;
+  private ServerWebSocket websocketMock;
   private final Map<String, JsonRpcMethod> methods = new HashMap<>();
 
   @Before
   public void before(final TestContext context) {
     vertx = Vertx.vertx();
```

Comment:
do we still need the vertx instance?

Reply:
no sure I full get this, but vertx is used in `handler.handle` method invocations